src/taxonomy/taxonomy-card/TaxonomyCard.jsx

```diff
@@ -101,7 +103,8 @@ const TaxonomyCard = ({ className, original, intl }) => {
 
   return (
     <>
-      <Card className={classNames('taxonomy-card', className)} data-testid={`taxonomy-card-${id}`}>
+      <Card isClickable className={classNames('taxonomy-card', className)} data-testid={`taxonomy-card-${id}`}>
```

Comment:
If you remove `isClickable` here, then the stretched-link be what gets followed when you click the card, or when you tab to the card and hit "enter".

Reply:
The `isClickable` adds a little hover effect to the card.
I think the following (1499bb50f3c7eb964eb4f2752f05d856eced1865) is the right way to do it (without the need of the `.stretched-link` class).